### ISSUE TITLE
The text of the message is not updated after editing if the current message is with an image

### ISSUE THREAD

**Issue by VolodLytvynenko** [status-im/status-react#15444]
Please see the comment from @ibrkhalil  before fixing this one https://github.com/status-im/status-mobile/pull/15190#issuecomment-1454684319

> Apparently, When the user edits an image with a photo .. The photo gets lost .. So the team have added a constaint to prevent editing a photo with text on status-go [here](https://github.com/status-im/status-go/blob/999d8c0ee0f2274c58b5a2de3beddab7feaabd16/protocol/messenger_messages.go#L19) .. Happy to investigate in a subsequent issue

#### Steps to reproduce:
1. Go to any chat
2. Send a message that contains some text and an image
3. Long press the message -> Tap 'edit'
4. Update text

#### Actual result:
The text is not updated

https://user-images.githubusercontent.com/52490791/222458375-d7d1ce76-6605-4688-9ccf-5de81c4d2d99.mp4

#### Expected result:
Text is updated

#### Env: 
nightly 2 Mar 2023

**Comment by yqrashawn** [comment]
error after edit message with image
```
[Thu Mar 23 2023 14:46:52.952]  ERROR    ERROR [status-im.chat.models.input:239] - failed to edit message  {:code -32000, :message "only text or emoji messages can be replaced"}
```

error is from status-go, code at https://github.com/status-im/status-go/blob/48eb7052848a17ab7f972112a3653f0b4cc8537a/protocol/messenger_messages.go#L36

**Comment by yqrashawn** [comment]
ah, ok, this should be a not yet implemented feature, which is 

allow user to edit both the text part and the image part of a message

related issue at https://github.com/status-im/status-mobile/issues/15298

design at https://www.figma.com/file/PPWkgOYlZZDxZv5SDGsZVV/Posts-%26-Attachments-for-Mobile?node-id=2790-304312&t=HTd3jBbAdpWReCMG-4

**Comment by yqrashawn** [comment]
Issue should be resolved once #15298 is implemented.

**Comment by yqrashawn** [comment]
After thinking it over again, we should disable the edit option for message with image until #15298 is implemented
